StitchCore/StitchCoreTests/StitchTestCase.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Utils.swift
+//  StitchCoreTests
+//
+//  Created by Jason Flax on 2/5/18.
+//  Copyright © 2018 MongoDB. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import PromiseKit
+import StitchLogger
+@testable import StitchCore
+
+internal extension StitchClient {
+    func registerAndConfirm(email: String,
+                            password: String,
+                            withHarness harness: TestHarness) -> Promise<Void> {
+        return self.register(email: email, password: password).then { _ in
+            return harness.app.userRegistrations.sendConfirmation(toEmail: email)
+        }.then { conf in
+            return self.emailConfirm(token: conf.token,
+                                     tokenId: conf.tokenId)
+        }.asVoid()
+    }
+}
+
+enum StitchTestError: Error {
+    case unknown
+}
+
+open class StitchTestCase: XCTestCase {
```

Comment:
[remark] Would be cool if you were allowed to be more restrictive here (i.e. make this internal)

Reply:
👍